### PR TITLE
import_image should return always true, regardless if tags changed

### PR DESCRIPTION
**Pull request by rcerven**
Signed-off-by: Robert Cerven <rcerven@redhat.com>

### CODE REVIEW

File: tests/test_core.py [hunk 1]
```diff
@@ -487,3 +489,26 @@ def test_use_service_account_token(self, kwargs, called):
         else:
             openshift_mock.never()
         Openshift(OAPI_PREFIX, API_VER, "/oauth/authorize", **kwargs)
+
+    @pytest.mark.parametrize('version', ["0.5.4", "1.0.4"])  # noqa
+    @pytest.mark.parametrize('make_different', [True, False])
```
Comment (rcerven) [inline tests/test_core.py]: sure

File: tests/test_core.py [hunk 2]
```diff
@@ -487,3 +489,26 @@ def test_use_service_account_token(self, kwargs, called):
         else:
             openshift_mock.never()
         Openshift(OAPI_PREFIX, API_VER, "/oauth/authorize", **kwargs)
+
+    @pytest.mark.parametrize('version', ["0.5.4", "1.0.4"])  # noqa
```
Comment (rcerven) [inline tests/test_core.py]: ahh nice, thanks

File: tests/mock_jsons/0.5.4/imagestream.json [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    },
+    "spec": {
+        "dockerImageRepository": "brew-pulp-docker01.web.qa.ext.phx1.redhat.com:8888/lucarval/rhel7",
+        "resources": {},
+        "tags": [
+            {
+                "annotations": null,
+                "from": {
+                    "kind": "DockerImage",
+                    "name": "brew-pulp-docker01.web.qa.ext.phx1.redhat.com:8888/lucarval/rhel7:7.2.lucarval-66"
```
Comment (twaugh) [inline tests/mock_jsons/0.5.4/imagestream.json]: Please change this to use example.com throughout.

File: tests/mock_jsons/0.5.4/imagestream.json [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+                "generation": 2,
+                "importPolicy": {
+                    "insecure": true
+                },
+                "name": "7.2.lucarval-68"
+            }
+        ]
+    },
+    "status": {
+        "dockerImageRepository": "172.30.202.13:5000/osbs-stage/lucarval-rhel7",
```
Comment (twaugh) [inline tests/mock_jsons/0.5.4/imagestream.json]: Please change this to use the 192.168.1.* network.

File: tests/mock_jsons/0.5.4/imagestream.json (hunk 3 above)
Comment (rcerven) [inline tests/mock_jsons/0.5.4/imagestream.json]: sure

File: tests/mock_jsons/0.5.4/imagestream.json (hunk 4 above)
Comment (rcerven) [inline tests/mock_jsons/0.5.4/imagestream.json]: sure

**Pull request merged**